MIGRATION.md

```diff
@@ -20,15 +20,15 @@ Depending on where you have your account, you have to visit separate sites to re
 
 1. Have Ferdi running on your system. (Even if you are on the initial "Get started" page, these instructions will work.)
 2. Go to `Help > Import/Export Configuration Data` which should open the corresponding url in your default browser.
-3. Click on `export your data to a ".ferdium-data" file`.
-4. Save the `export.fedi-data` file to anywhere you'd like for safe-keeping/backup.
+3. Click on `export your data to a ".ferdi-data" file`.
+4. Save the `export.ferdi-data` file to anywhere you'd like for safe-keeping/backup.
 
 ## Importing
 
-1. Have Ferdi running on your system.
+1. Have Ferdium running on your system.
```

Comment:
Importing is only supported into Ferdium - not Ferdi. Please revert the changes in the `import` section.

Reply:
maybe it can apply on to this line - if the thought process is for "export from ferdi, import into ferdium".

Reply:
This doc is about migration, titled MIGRATION.md. My thought process, therefore, is "Migration from Ferdi to Ferdium".

The introduction also states this:

> This guide is here to show you how to export your current Ferdi setup from the online server or from an accountless session for safe-keeping, backup or transferring to a new instance using only a browser.

For importing Ferdium must be running since I am importing the Ferdi data into Ferdium. Or am I mistaken?

---

> Importing is only supported into Ferdium - not Ferdi. Please revert the changes in the import section.

Does not make any sense to me; what exactly do you mean?

Reply:
sorry - i misread your change (in reverse order - ie Ferdium to Ferdi)